hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/common/volume/TestReservedVolumeSpace.java

```diff
@@ -95,15 +95,18 @@ public void testVolumeCapacityAfterReserve() throws Exception {
         HDDS_DATANODE_DIR_DU_RESERVED_PERCENT_DEFAULT);
 
     long volumeCapacity = hddsVolume.getCurrentUsage().getCapacity();
+    long volumeAvailable = hddsVolume.getCurrentUsage().getAvailable();
     VolumeUsage usage = hddsVolume.getVolumeInfo().get().getUsageForTesting();
 
     //Gets the actual total capacity
     long totalCapacity = usage.realUsage().getCapacity();
+    long totalAvailable = usage.realUsage().getAvailable();
```

Comment:
Similarly here, please assign `usage.realUsage()` to variable.

Reply:
Updated, PTAL.